Modules/Scripted/ExtensionWizard/ExtensionWizard.py

```diff
@@ -384,25 +387,26 @@ def loadModules(path=None, depth=1, modules=None, parent=None):
 
                 if len(failed):
                     if len(failed) > 1:
-                        text = "The following modules could not be registered:"
+                        text = _("The following modules could not be registered:")
```

Comment:
Change it to something like `{count} modules could not be registered.` (nothing is added after the `:`, so it does not make sense to end the text with that).